BankAccounts.rb

```diff
@@ -19,15 +19,18 @@ def assign_owner(person)
     def withdrawl(amnt_withdrawn)
       if amnt_withdrawn > @balance
         puts "You can't take out more than #{@balance}"
+        return @balance
       else
       @balance = @balance - amnt_withdrawn # Return the updated account balance.
       puts "Your current balance is: #{@balance}" # The updated balance
       end
+      return @balance
     end
 
     def deposit(amnt_deposited)
       @balance = @balance + amnt_deposited #Return the updated account balance.
       puts "Your current balance is: #{@balance}"
+      return @balance
     end
 
 # This method returns all of the accounts in one array as Account instances.
```

Comment:
I can't comment on the lines below, but I would recommend using a more "useful" variable name rather than `l` in your `each` loop below in lines 43-47